Dockerfile

```
@@ -36,6 +36,7 @@ COPY Gemfile ${DEPS_HOME}/Gemfile
 COPY Gemfile.lock ${DEPS_HOME}/Gemfile.lock
 
 RUN gem install bundler
+RUN bundle config --local build.sassc --disable-march-tune-native
```

Would setting the an environment variable, as outlined in https://github.com/sass/sassc-ruby/issues/146#issuecomment-542288556, be a neater solution?

```suggestion
ENV BUNDLE_BUILD__SASSC=--disable-march-tune-native
```